star-rx72n-firmware/lib/rx_drv8243/src/rx_drv8243.c

```
@@ -437,3 +517,555 @@ static rx_err_t internal_drv8243_configure_fault_pin(const rx_drv8243_handle_t*
+
+  /* Write back */
+  err = internal_drv8243_spi_write_reg(handle, k_drv8243_reg_config3, config3);
+  if (err != k_rx_ok) {
+    return err;
+  }
+
+  return k_rx_ok;
+}
+
+rx_err_t rx_drv8243_spi_set_itrip(rx_drv8243_handle_t*        handle,
+                                  const drv8243_itrip_level_t level,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_drv8243_spi_set_itrip' of similar type ('const int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)